alerts/cluster-monitoring-operator/ClusterMonitoringOperatorDeprecatedConfig.md

```diff
@@ -0,0 +1,33 @@
+# ClusterMonitoringOperatorDeprecatedConfig
+
+## Meaning
+
+This alert fires when a deprecated config is used in the config map `openshift-monitoring/cluster-monitoring-config`.
+
+## Impact
+
+Using a deprecated config is not recommended because the config has no effect and
+it may cause `cluster-monitoring-operator` go in degraded state when the config
```

Comment:
Before we remove the field, we'll block the upgrades using the Upgradeable condition.

```suggestion
it may cause the Cluster Monitoring operator `Upgradeable` condition to become `False`  in a future OCP release.
```